Create ui-c8y-1023-5-0-allow-skipping-sso-redirect-and-login-with-platform-credentials-10431.md (#3963)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1023-5-0-allow-skipping-sso-redirect-and-login-with-platform-credentials-10431.md

---
date: ""

> title: Allow skipping SSO redirect and login with platform credentials

Reviewer: ```suggestion
title: New option for skipping SSO redirect and showing standard platform login
```

product_area: Application enablement & solutions
change_type:
  - value: change-QHu1GdukP
    label: Feature
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-65303
version: 1023.5.0
---

> Co-authored-by: Tristan Bastian <tristan.bastian@cumulocity.com>

Reviewer 2: ```suggestion
In some scenarios, users may prefer to log in directly with their platform credentials instead of being redirected to Single Sign-On (SSO).
To address this need, a new feature has been implemented that allows users to skip the automatic SSO redirect and log in using their platform credentials instead.

A new `skipSSORedirect` query parameter has been introduced for this usecase.
```

---
date: ""
title: Allow skipping SSO redirect and login with platform credentials
product_area: Application enablement & solutions
change_type:
  - value: change-QHu1GdukP
    label: Feature
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-65303
version: 1023.5.0
---
In some scenarios, users may prefer to log in directly with their platform credentials instead of being redirected to Single Sign-On (SSO).
To address this need, a new feature has been implemented that allows users to skip the automatic SSO redirect and log in using their platform credentials instead.

> A new `skipSSORedirect` query parameter has been introduced for this usecase.

Reviewer 3: ```suggestion
To support a broader range of authentication workflows, the Web SDK now introduces the `skipSSORedirect` query parameter. When enabled, this option skips the default single sign-on (SSO) redirect and displays the standard platform login interface instead. This enhancement provides greater control over authentication behavior, making it easier to support environments where direct platform login is preferred or where SSO is unnecessary.
```
